mteb/tasks/Retrieval/fra/FQuADRetrieval.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class FQuADRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLITS = ["test", "validation"]
+
+    metadata = TaskMetadata(
+        name="FQuADRetrieval",
+        description="This dataset has been built from the French SQuad dataset.",
+        reference="https://huggingface.co/datasets/manu/fquad2_test",
+        dataset={
+            "path": "manu/fquad2_test",
+            "revision": "5384ce827bbc2156d46e6fcba83d75f8e6e1b4a6",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=_EVAL_SPLITS,
+        eval_langs=["fra-Latn"],
+        main_score="map",
+        date=("2019-11-01", "2020-05-01"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Article retrieval"],
+        license="apache-2.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@misc{dhoffschmidt2020fquad,
+      title={FQuAD: French Question Answering Dataset}, 
+      author={Martin d'Hoffschmidt and Wacim Belblidia and Tom Brendlé and Quentin Heinrich and Maxime Vidal},
+      year={2020},
+      eprint={2002.06071},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}""",
```

Comment:
```suggestion
        bibtex_citation="""@inproceedings{dHoffschmidt2020FQuADFQ,
  title={FQuAD: French Question Answering Dataset},
  author={Martin d'Hoffschmidt and Maxime Vidal and Wacim Belblidia and Quentin Heinrich and Tom Brendl'e},
  booktitle={Findings},
  year={2020},
  url={https://api.semanticscholar.org/CorpusID:211126910}
}""",
```
I think this citation would be better for our upcoming paper 🙂

Reply:
btw isn't it this one: https://aclanthology.org/2022.lrec-1.237/ ?

Reply:
The V2 only contains adversarial samples so theoretically it's the V1 here since we don't have any !